docs/swagger.yaml

```diff
@@ -71,16 +71,45 @@ paths:
           name: Accept
           required: true
           description: Requested image format
-        - schema:
-            type: string
-          in: header
-          name: Ocp-Apim-Subscription-Key
-          description: Subscription Key
-          required: true
+        - $ref: '#/components/parameters/Ocp-Apim-Subscription-Key'
       tags:
         - QR
       security:
         - BearerToken: []
+  /v1/merchantRedirect:
+    post:
+      summary: CreateMerchantRedirectQr
+      operationId: CreateMerchantRedirectQr
+      responses:
+        '201':
+          description: Created
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/MerchantRedirectQrResponse'
+              examples:
+                example-1:
+                  value:
+                    id: '{{uniqueId}}'
+                    qrUrl: 'https://qr.vipps.no/generate/qr.png?uri=https://qr-test.vipps.no/c/VFbOmash'
+      description: Generate a QR qith custom target url
+      tags:
+        - MerchantRedirect
+      parameters:
+        - $ref: '#/components/parameters/Ocp-Apim-Subscription-Key'
```

Comment:
you are missing the accept headers here. Look at how its done on the existing endpoint